ArduCopter/mode_auto.cpp

```diff
@@ -876,7 +876,7 @@ void Copter::ModeAuto::land_run()
     // set motors to full range
     motors->set_desired_spool_state(AP_Motors::DESIRED_THROTTLE_UNLIMITED);
     
-    land_run_horizontal_control();
+    land_run_horizontal_control(auto_yaw.mode()==AUTO_YAW_FIXED, auto_yaw.yaw());
```

Comment:
Ah.. I see why we using this. But still my question lies, how were we able to compile the method without the params in the first place, as on looking at the repo I couldn't see any polymorphic function there (without the params)?

Reply:
it takes optional parameters (ie parameters with default values)